wd.sh

```diff
@@ -290,16 +292,17 @@ wd_show()
             wd_print_msg "$WD_GREEN" "Warp point: ${WD_GREEN}$name_arg${WD_NOC} -> $points[$name_arg]"
         fi
     else
+        [[ $extglob ]] && setopt noextendedglob
         # hax to create a local empty array
         local wd_matches
         wd_matches=()
         # do a reverse lookup to check whether PWD is in $points
         PWD="${PWD/$HOME/~}"
-        if [[ ${points[(r)$PWD]} == "$PWD" ]]
+        if [[ ${points[(r)${PWD}]} == ${PWD} ]]
```

Comment:
No, that's an artifact if my prodding around, trying to make sense of the situation - feel free to adept the code to your liking, otherwise I'll revert the unnecessary changes later

Reply:
Thanks!